src/guiguts/file.py

```diff
@@ -563,7 +563,7 @@ def do_goto_image(self, image_num: Optional[str]) -> None:
         if image_num is not None:
             try:
                 index = maintext().rowcol(page_mark_from_img(image_num))
-            except tk._tkinter.TclError:  # type: ignore[attr-defined]
```

Comment:
I believe this could be rewritten as `tk.TclError`; functionally the same, but it would stop the linter complaint.

```
>>> import tkinter as tk
>>> tk._tkinter.TclError is tk.TclError
True
```

Reply:
Thanks - you're right - not sure why I ended up making it more complicated.